packages/templates/clients/websocket/javascript/package.json

```
@@ -67,6 +67,15 @@
         "description": "The name of the generated client file",
         "required": false,
         "default": "client.js"
+      },
+      "appendClientSuffix": {
+        "description": "Add Client at the end of the class name",
```

```suggestion
        "description": "Add 'Client' suffix at the end of the class name",
```
I think the original version could be confuse people in the sense that they may not quickly understand that "Client" here refers to the english word, and not a code object.
Same for the python package.